src/lualib/Map.ts

```diff
@@ -130,7 +128,7 @@ Map = class Map<K, V> {
     }
 
     public keys(): IterableIterator<K> {
-        const nextKey = this.nextKey;
+        const { nextKey } = this;
```

Comment:
Does eslint force this? I am not sure about this rule actually.

Reply:
That's a [`prefer-destructuring`](https://eslint.org/docs/rules/prefer-destructuring) rule. I don't really mind disabling it.